x/farming/keeper/staking.go

```diff
@@ -60,6 +60,15 @@ func (k Keeper) IterateStakingsByFarmer(ctx sdk.Context, farmerAcc sdk.AccAddres
 	}
 }
 
+func (k Keeper) GetAllStakingsByFarmer(ctx sdk.Context, farmerAcc sdk.AccAddress) sdk.Coins {
```

Comment:
The name should be something like `GetAllStakedCoinsByFarmer` or `GetStakedCoinsByFarmer` since `GetAllStakingsByFarmer` seems to return `[]types.Staking` instead of `sdk.Coins`.